uds/can/flow_control.py

```diff
@@ -57,9 +57,9 @@ class CanFlowStatus(ValidatedEnum, NibbleEnum):
 
     ContinueToSend: "CanFlowStatus" = 0x0  # type: ignore
     """Asks to resume Consecutive Frames transmission."""
-    Wait: "CanFlowStatus" = 0x1  # type: ignore  # noqa: F841
+    Wait: "CanFlowStatus" = 0x1  # type: ignore
```

Comment:
`from __future__ import annotations`

What do you think about using "annotations" instead of manually adding forward references and suppressing type errors with `# type: ignore`?

Reply:
I need more information, I do not know how it would help with Enum members annotation.

Reply:
Ok, sorry, its not working the way I thought it is, my bad. But anyway, why do you need to add this typing here?

Reply:
I need it for dynamic typing checking.
- If I put it here, I inform [MyPy](https://mypy.readthedocs.io/en/stable/getting_started.html) about the actual type of these attributes and MyPy would be able to continue with dynamic checking properly later on.
- If I suppress it when parameter when passed as argument, then I have no type checking for the function that takes enum members as arguments.

In my opinion this is the best solution we could get.

Reason:
MyPy is unable right now to detect enum members (they have special implementation in `__new__` method which is kinda unsual and hidden).